learn/experimental/vector_search.mdx

```diff
@@ -140,6 +140,35 @@ curl -X POST -H 'content-type: application/json' \
 ]'
 ```
 
+### Distribution shift
+
+Hybrid search results do not always use the full vector space uniformly, which may impact relevancy. Use `distributionShift` when configuring an embedder to correct result distribution with an affine transformation:
```

Comment:
Can we add [more context](https://meilisearch.notion.site/v1-8-AI-search-API-usage-135552d6e85a4a52bc7109be82aeca42#d652adc0890445658aaf36352dbc8802) to what those values mean, please?